linguistics/src/main/java/com/yahoo/language/significance/SignificanceModelRegistry.java

```diff
@@ -0,0 +1,11 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.language.significance;
```

Comment:
Add a `package-info.java` with the two annotations `@PublicApi` and `@ExportPackage` to the `com.yahoo.language.significance` package.